src/lib/analysers/ProtestsPretrained/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytorch
```

Comment:
This should be 'torch'. When building a Docker image, mtriage collects all the requirements from components' requirements.txt files and installs them- and the correct way to install pytorch through pip is `pip install torch`.

Can you give a version as well? i.e. `torch==1.4`.

Is `torchvision` also required? If so it needs to also be listed here.

Reply:
I added the main requirements, indeed including torchvision. I don't think the newest versions of torch & co are needed, I could test with other versions if needed.